### PR TITLE
NATIVE-580: Only deliver every two weeks to beta

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **NATIVE**.

### CODE REVIEW

File: .circleci/src/@workflows.yml [hunk 1]
```diff
@@ -60,10 +60,10 @@ workflows:
           requires:
             - bump_version
 
-  weekly_development_delivery:
+  bi_weekly_development_delivery:
     triggers:
       - schedule:
-          cron: "0 3 * * 4" # For sanity checks for this syntax: https://crontab.guru/
+          cron: "0 3 8,22 * *" # For sanity checks for this syntax: https://crontab.guru/
```
Comment (Taggotty) [inline .circleci/src/@workflows.yml:66]: Why is this value different from the one above?

Comment (Taggotty) [reply .circleci/src/@workflows.yml:66]: And why do we need to define 2 schdules? One in the workflow and one in the task itself?

Comment (klinzo) [reply .circleci/src/@workflows.yml:66]: What value above do you mean? The `config.yml` is put together by the other config files, e.g. this `@workflow.yml` file.

There are two schedules: Development delivery and production delivery.

**Comment by klinzo** [comment]
@Taggotty anything to add or can I merge this?

**Pull request merged**